src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/clustering/SomaticClusteringModel.java

```diff
@@ -101,8 +101,8 @@ public void record(int[] tumorADs, final double[] tumorLogOdds, final List<Doubl
         // get all alt allele indexes for symbolic alleles
         List<Integer> symIndexes = new IndexRange(0, vc.getNAlleles()-1).filter(n -> vc.getAlternateAllele(n).isSymbolic());
 
-        // set tumorAD to 0 for symbolic alleles so it won't contribute to overall AD
-        symIndexes.forEach(i -> tumorADs[i] = 0);
+        // set tumorAD to 0 for symbolic alleles so it won't contribute to overall AD, ADs include reference allele so add one to the index
+        symIndexes.forEach(i -> tumorADs[i+1] = 0);
```

Comment:
I just want to make sure that this can't try to read past the end of the array. I think that the non-ref allele is always at the end and as long as that is not a symbolic allele, I think you should be ok. Just want to make sure that's correct.

Reply:
It should be correct, but would it be better to put an Assert in for the lengths of the Arrays?